grails-app/taglib/org/broadinstitute/orsp/AuthTagLib.groovy

```diff
@@ -61,6 +61,25 @@ class AuthTagLib {
         }
     }
 
+    /**
+     * Renders the tag body when the user does not have ro_admin role
```

Comment:
nit: I think this is the opposite of what the function is doing. It looks like it is printing the body content when the user **does have the role in question**

Reply:
yes, comments are the ooposite in isViewer and isNotViewer. Will fix it.